addons/website_mass_mailing/models/website_mass_mailing.py

```diff
@@ -13,4 +14,4 @@ def _default_popup_content(self):
 
     mailing_list_id = fields.Many2one('mailing.list')
     website_id = fields.Many2one('website')
-    popup_content = fields.Html(string="Website Popup Content", default=_default_popup_content, translate=True, sanitize=False)
+    popup_content = fields.Html(string="Website Popup Content", default=_default_popup_content, translate=html_translate, sanitize=False)
```

Comment:
I think you will kill any translation that exist for every database of odoo, you should not change it.